MinPlatformPkg/Test/Library/TestPointLib/StandaloneMmTestPoint.h

```diff
@@ -0,0 +1,64 @@
+/** @file
+
+  Copyright (c) 2017, Intel Corporation. All rights reserved.<BR>
+  SPDX-License-Identifier: BSD-2-Clause-Patent
+
+**/
+
+#ifndef _STANDALONE_MM_TEST_POINT_H_
+#define _STANDALONE_MM_TEST_POINT_H_
+
+#include <PiMm.h>
+#include <Library/BaseLib.h>
+#include <Library/BaseMemoryLib.h>
+#include <Library/MemoryAllocationLib.h>
+#include <Library/MmServicesTableLib.h>
+#include <Library/StandaloneMmMemLib.h>
+#include <Library/DebugLib.h>
+#include <Library/TestPointLib.h>
+
+#include <Protocol/AdapterInformation.h>
+
+#define TEST_POINT_AIP_PRIVATE_SIGNATURE  SIGNATURE_32('T', 'S', 'P', 'T')
```

Comment:
This file has more content I wouldn't expect to be duplicated across various header files. For example, should `TEST_POINT_AIP_PRIVATE_SIGNATURE` every vary depending on the phase?

I guess it's okay for what is already there for DXE & SMM, but there might be an opportunity to reuse content across MM where it doesn't need to differ.

Reply:
Removed redundancies.